content/en/docs/collector/quick-start.md

```diff
@@ -60,8 +60,10 @@ preferred shell.
 3. Launch the Collector:
 
    ```sh
+   # 4317 for gRPC, 4318 for HTTP, 55679 for ZPages
```

Comment:
I still feel it better to just drop the comment, but will let other @open-telemetry/docs-approvers chime in. If we keep it, let's be clear that this comment is referring to port numbers.

Reply:
Can we move that comment into the text above, like "Launch the Collector, listening on ports 4317 (for OTLP grPC), 4318 (for OTLP HTTP) and 55679 (for ZPages):"

Reply:
Either way works for me! I updated the PR to use the text @svrnm suggested in: c9d08a4e